modules/openapi-generator/src/main/resources/typescript-rxjs/runtime.mustache

```diff
@@ -97,6 +97,7 @@ export class BaseAPI {
             method: requestOpts.method,
             headers: requestOpts.headers,
             body: requestOpts.body instanceof FormData ? requestOpts.body : JSON.stringify(requestOpts.body),
+            responseType: requestOpts.responseType || 'json'
```

Comment:
See
https://github.com/OpenAPITools/openapi-generator/blob/2c4fdd079207374822269b58b4f7a93378456b3c/modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache#L139
and
https://github.com/OpenAPITools/openapi-generator/blob/2c4fdd079207374822269b58b4f7a93378456b3c/modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache#L339-L341
to have an example how the response type is determined in the angular generator.

Reply:
Thanks for the feedback. I will look into this but I have a feeling that these are the missing pieces.